core/plugins/org.fusesource.ide.camel.model.service.impl.v2210fuse740039redhat00001/build.properties

```diff
@@ -0,0 +1,16 @@
+source.. = src/
+bin.includes = META-INF/,\
+               .,\
+               libs/camel-blueprint-2.21.0.fuse-731003-redhat-00003.jar,\
```

Comment:
the build.properties contains 7.3 values instead of 7.4

Reply:
pushed the fix on the branch of the PR https://github.com/lhein/jbosstools-fuse/commit/920478f2a8a9724189a1760f9225d0c1492db75e but seems GitHub is not noticing it...

Reply:
it has finally detected the change right after my last comment

Reply:
thanks Aurelien